index.js

```diff
@@ -231,7 +231,10 @@ async function requestUpholdTickers (currency) {
       ask
     }) => {
       if (currency !== 'USD') {
-        const alt = pair.slice(3)
+        let alt = pair.slice(3)
+        if (alt[0] === '-') {
```

Comment:
What are the values of the pair are we trying to handle in this case?

Reply:
recently they introduced ones with `-` between the pairs. where normally they were jammed together: `USDDAI` now some of them are split by a `-` e.g. `USD-DAI`

Reply:
endpoint is here: https://api.uphold.com/v0/ticker/USD